src/tools/wasm-opt.cpp

```diff
@@ -48,9 +49,22 @@ int main(int argc, const char* argv[]) {
            })
       .add("", "-O", "execute default optimization passes",
            Options::Arguments::Zero,
-           [&passes](Options*, const std::string&) {
+           [&passes, &passOptions](Options*, const std::string&) {
+             passOptions.setDefaultOptimizationOptions();
              passes.push_back("O");
            })
+      .add("", "-Os", "execute default optimization passes, focusing on code size",
+           Options::Arguments::Zero,
+           [&passes, &passOptions](Options*, const std::string&) {
+             passOptions.setDefaultOptimizationOptions();
+             passOptions.shrinkLevel = 1;
+             passes.push_back("O");
+           })
+      .add("--shrink-level", "-s", "How much to focus on shrinking code size",
```

Comment:
Do we really want 2 separate knobs for this? What happens if someone says `O3` but also `-s1`? Those are contradictory. Especially if we are going to have `Os` and/or `Oz` then we don't also need a separate shrink argument, do we?